tests/bluetooth/host/common/defaults_cmake.txt

```diff
@@ -0,0 +1,55 @@
+#
+# Common include directories and source files for bluetooth unit tests
+#
+
+include_directories(
+  ${CMAKE_CURRENT_SOURCE_DIR}/
+  ${CMAKE_CURRENT_SOURCE_DIR}/src
+  ${ZEPHYR_BASE}/tests/bluetooth/host/mock_host/include
+  ${ZEPHYR_BASE}/tests/bluetooth/host/common/include
+  ${ZEPHYR_BASE}/include/zephyr/net
+  ${ZEPHYR_BASE}/include/zephyr/bluetooth
+  ${ZEPHYR_BASE}/subsys/bluetooth
+  ${ZEPHYR_BASE}/subsys/bluetooth/host
+  ${ZEPHYR_BASE}/subsys/testsuite/ztest/include/zephyr
+)
+
+FILE(GLOB host_sources
+  ${ZEPHYR_BASE}/subsys/bluetooth/host/buf.c
+)
+
+FILE(GLOB mock_sources
+  ${ZEPHYR_BASE}/tests/bluetooth/host/mock_host/src/net/buf.c
+  ${ZEPHYR_BASE}/tests/bluetooth/host/mock_host/src/iso.c
+  ${ZEPHYR_BASE}/tests/bluetooth/host/mock_host/src/hci_core.c
+  ${ZEPHYR_BASE}/tests/bluetooth/host/mock_host/src/assert.c
+)
+
+add_definitions(-include kconfig.h)
+
+if(COMPILE_PARAM_1)
+  add_compile_definitions(${COMPILE_PARAM_1})
+endif()
+
+if(COMPILE_PARAM_2)
+  add_compile_definitions(${COMPILE_PARAM_2})
+endif()
+
+if(COMPILE_PARAM_3)
+  add_compile_definitions(${COMPILE_PARAM_3})
+endif()
+
+if(COMPILE_PARAM_4)
+  add_compile_definitions(${COMPILE_PARAM_4})
+endif()
+
+if(COMPILE_PARAM_5)
+  add_compile_definitions(${COMPILE_PARAM_5})
+endif()
+
+if(KCONFIG_OVERRIDE_FILE)
+  add_definitions(-include ${KCONFIG_OVERRIDE_FILE})
+endif()
+
+add_definitions(-include ztest.h)
+add_definitions(-include soc.h)
```

Comment:
Please instead add `#include`s to each file that needs this.

Reply:
It seems that some other files (source and header files) that don't exist in the project directory depends on the first one `add_definitions(-include ztest.h)` and causes the compilation process fails.

I removed the second one as it doesn't have any effect